src/vivarium_gates_child_iv_iron/constants/results.py

```diff
@@ -29,28 +29,50 @@
-STATE_PERSON_TIME_COLUMN_TEMPLATE = '{STATE}_person_time_in_{YEAR}_among_{SEX}_in_age_group_{AGE_GROUP}'
-TRANSITION_COUNT_COLUMN_TEMPLATE = '{TRANSITION}_event_count_in_{YEAR}_among_{SEX}_in_age_group_{AGE_GROUP}'
+DEATH_COLUMN_TEMPLATE = ('death_due_to_{CAUSE_OF_DEATH}_year_{YEAR}_sex_{SEX}_age_{AGE_GROUP}'
+                         '_wasting_state_{CGF_RISK_STATE}_stunting_state_{CGF_RISK_STATE}')
+YLLS_COLUMN_TEMPLATE = ('ylls_due_to_{CAUSE_OF_DEATH}_year_{YEAR}_sex_{SEX}_age_{AGE_GROUP}'
+                        '_wasting_state_{CGF_RISK_STATE}_stunting_state_{CGF_RISK_STATE}')
+YLDS_COLUMN_TEMPLATE = ('ylds_due_to_{CAUSE_OF_DISABILITY}_year_{YEAR}_sex_{SEX}_age_{AGE_GROUP}'
+                        '_wasting_state_{CGF_RISK_STATE}_stunting_state_{CGF_RISK_STATE}')
+DIARRHEA_STATE_PERSON_TIME_COLUMN_TEMPLATE = (
+    'diarrheal_diseases_{DIARRHEA_STATE}_person_time_year_{YEAR}_sex_{SEX}_age_{AGE_GROUP}'
+    '_wasting_state_{CGF_RISK_STATE}_stunting_state_{CGF_RISK_STATE}')
+LRI_STATE_PERSON_TIME_COLUMN_TEMPLATE = (
+    'lower_respiratory_infections_{LRI_STATE}_person_time_year_{YEAR}_sex_{SEX}_age_{AGE_GROUP}'
+    '_wasting_state_{CGF_RISK_STATE}_stunting_state_{CGF_RISK_STATE}')
+MEASLES_STATE_PERSON_TIME_COLUMN_TEMPLATE = (
+    'measles_{MEASLES_STATE}_person_time_year_{YEAR}_sex_{SEX}_age_{AGE_GROUP}'
+    '_wasting_state_{CGF_RISK_STATE}_stunting_state_{CGF_RISK_STATE}')
+DIARRHEA_TRANSITION_COUNT_COLUMN_TEMPLATE = (
+    'diarrheal_diseases_{DIARRHEA_TRANSITION}_event_count_year_{YEAR}_sex_{SEX}_age_{AGE_GROUP}'
+    '_wasting_state_{CGF_RISK_STATE}_stunting_state_{CGF_RISK_STATE}')
+LRI_TRANSITION_COUNT_COLUMN_TEMPLATE = (
+    'lower_respiratory_infections_{LRI_TRANSITION}_event_count_year_{YEAR}_sex_{SEX}_age_{AGE_GROUP}'
+    '_wasting_state_{CGF_RISK_STATE}_stunting_state_{CGF_RISK_STATE}')
+MEASLES_TRANSITION_COUNT_COLUMN_TEMPLATE = (
+    'measles_{MEASLES_TRANSITION}_event_count_year_{YEAR}_sex_{SEX}_age_{AGE_GROUP}'
+    '_wasting_state_{CGF_RISK_STATE}_stunting_state_{CGF_RISK_STATE}')
 
 COLUMN_TEMPLATES = {
     'population': TOTAL_POPULATION_COLUMN_TEMPLATE,
     'deaths': DEATH_COLUMN_TEMPLATE,
     'ylls': YLLS_COLUMN_TEMPLATE,
     'ylds': YLDS_COLUMN_TEMPLATE,
-    'state_person_time': STATE_PERSON_TIME_COLUMN_TEMPLATE,
-    'transition_count': TRANSITION_COUNT_COLUMN_TEMPLATE,
+    'diarrhea_state_person_time': DIARRHEA_STATE_PERSON_TIME_COLUMN_TEMPLATE,
+    'measles_state_person_time': MEASLES_STATE_PERSON_TIME_COLUMN_TEMPLATE,
+    'lri_state_person_time': LRI_STATE_PERSON_TIME_COLUMN_TEMPLATE,
+    'diarrhea_transition_count': DIARRHEA_TRANSITION_COUNT_COLUMN_TEMPLATE,
+    'measles_transition_count': MEASLES_TRANSITION_COUNT_COLUMN_TEMPLATE,
+    'lri_transition_count': LRI_TRANSITION_COUNT_COLUMN_TEMPLATE,
 }
 
-
 NON_COUNT_TEMPLATES = [
 ]
 
 POP_STATES = ('living', 'dead', 'tracked', 'untracked')
 SEXES = ('male', 'female')
-YEARS = tuple(range(2022, 2026))
+YEARS = tuple(range(2022, 2025))
```

Comment:
why did this change from 2026 to 2025?

Reply:
This was for the run up to 2024. It doesn't really need to change, but should it have originally been 2027 because the right range isn't exclusive and we would normally have data up to 2026?

Reply:
Ok sounds good.